Pod/Classes/SharedTypes.swift

```diff
@@ -67,7 +65,7 @@ public enum TargetView {
             #elseif os(OSX)
                 return gestureRecognizer.view?.window?.contentView
             #endif
-        case .this(let view):
+        case let .this(view):
```

Comment:
Did you made this change manually or you run something like SwiftLint or SwiftFormat?
If so, what configuration file did you use? It might be interesting to include it in the repository 🙂

Reply:
I did this manually, but there is a rule for multiple pattern matching that is already used in .swiftlint.yml - https://realm.github.io/SwiftLint/pattern_matching_keywords.html. So, we can follow this code style.

Reply:
I forgot to mention it in my first comment but I was also thinking about all `return` removed from single statement function bodies.

Reply:
[Implicit returns from single-expression functions](https://github.com/apple/swift-evolution/blob/master/proposals/0255-omit-return.md) - is feature of swift 5.1

Reply:
Looks like swiftlint doesn't work correctly because of the project path used -> fixed. Add implicit return rule - https://realm.github.io/SwiftLint/implicit_return.html